pkg/metrics/tags_test.go

```diff
@@ -89,7 +89,7 @@ func TestContexts(t *testing.T) {
 		},
 	}, {
 		name: "revision context",
-		ctx: mustCtx(t, func() (context.Context, error) {
```

Comment:
Should we replace other `mustCtx` usage with `purge`?

Reply:
need mustCtx for the other ones because they do return an error :(